### ISSUE TITLE
Why propel3?

### ISSUE THREAD

**Issue by taoso** [propelorm/Propel3#52]
Is there any guy can offers the reason to create Propel3 and abandon propel2? Thanks.

**Comment by gossi** [comment]
Think about it as PHP6 vs. PHP7.

**Comment by marcj** [comment]
Exactly, Propel2 is already several years old and started as pure active-record (which was at that time already almost dead, especially when you focus on active-record only), so we decided to name it Propel3 since its a pure data-mapper now and a completely new code base for the persisting part, as v2 is still heavily in use by the community and I won't break thousands of applications by increasing the alpha version.

**Comment by kinekt4** [comment]
"Propel3 is an open-source Object-Relational Mapping (ORM) for modern PHP 7.1+"

Does/will it work on PHP 5.*?

**Comment by gossi** [comment]
no.
